frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverview.tsx

```diff
@@ -28,11 +29,16 @@ type MonitoringOverviewProps = {
   resource: K8sResourceKind;
   pods?: PodKind[];
   resourceEvents?: FirehoseResult<EventKind[]>;
+  monitoringAlerts: Alert[];
 };
 
 const MonitoringOverview: React.FC<MonitoringOverviewProps> = (props) => {
-  const { resource, pods, resourceEvents } = props;
-  const [expanded, setExpanded] = React.useState(['metrics']);
+  const { resource, pods, resourceEvents, monitoringAlerts } = props;
+  const firingAlerts = getFiringAlerts(monitoringAlerts);
+  const [expanded, setExpanded] = React.useState([
+    'metrics',
+    firingAlerts.length > 0 && 'monitoring-alerts',
```

Comment:
This has the correct behavior but it will add `false` to the array which isn't logically correct.
```suggestion
    ...(firingAlerts.length > 0 ?  ['monitoring-alerts'] : []),
```